code/game/machinery/autolathe/mechfab.dm

```diff
@@ -81,3 +81,48 @@
 		MATERIAL_PLASTIC = 30,
 		MATERIAL_GLASS = 30,
 		)
+
+/obj/machinery/autolathe/mechfab/grayson
+	name = "Greyson fabricator"
+	desc = "A machine used for construction of Greyson designs."
+	icon_state = "greyson_nade_printer"
+	circuit = /obj/item/circuitboard/mechfab/grayson
+
+	build_type = GREYSONFAB
+	storage_capacity = 240
+	speed = 0.2 //takes a bit of time
```

Comment:
That is 10 time slower than the protolathe and 15 time slowers than the exofab. Why?

If you didn't want people to spam the roomba grenades, why not just make *those* take a while to print instead of slowing the whole thing down ?

Not to mention that it doesn't help against simply putting the designs on a disk and printing it on the lathe.